test/testInlineStylesheets/main.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
I don't think this file is used? If not, please remove. 
And if you want to, also remove test/testInlineScriptTags/main.js (also unused. I think this was a copy paste leftover).

Reply:
I removed main.css and there were two html files for inlineScriptTags, so I removed myHtml.html and only left one index.html (see d476ffe)